### PR TITLE
vikunja-frontend: 0.19.1 -> 0.20.2

### PR DESCRIPTION
**Pull request by kolaente**
###### Description of changes

Vikunja frontend 0.20, 0.20.1, 0.20.2 release. Changelogs:
*  https://kolaente.dev/vikunja/frontend/src/branch/main/CHANGELOG.md
* https://vikunja.io/blog/2022/11/vikunja-0.20.1-was-released/
* https://vikunja.io/blog/2022/12/vikunja-frontend-0.20.2-was-released/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/web-apps/vikunja/frontend.nix [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 stdenv.mkDerivation rec {
   pname = "vikunja-frontend";
-  version = "0.19.1";
+  version = "0.20.0";
   src = fetchurl {
     url = "https://dl.vikunja.io/frontend/${pname}-${version}.zip";
-    sha256 = "sha256-Kf55M1m/NBQhgaul/4seDMdPFU8jhgOwTNAzdgVg2OQ=";
+    sha256 = "sha256-1dZ02kFnOPrT6YfKdo8u3Buc0y9pFAVgb/b2tT+EL3c=";
```
[the remark this answers is not in the document]
Comment (kolaente) [reply pkgs/servers/web-apps/vikunja/frontend.nix:8]: That's... interesting. I'll push a fix.

[the remark this answers is not in the document]
Comment (kolaente) [reply pkgs/servers/web-apps/vikunja/frontend.nix]: Should be fine now.

**Comment by kolaente** [comment]
From my pov it's only missing a review.

**Comment by mweinelt** [comment]
```
error: hash mismatch in fixed-output derivation '/nix/store/jza080g3ipgirpb15hyzhwz4nx26xgfb-vikunja-frontend-0.20.1.zip.drv':
         specified: sha256-iKneTo+WVllmCRcJevbQFXfwSBhCY/V+aqTwfoT1XIc=
            got:    sha256-iPiAQrkAK9tODxY+2fjOBEjXVflW9CnCLJCSngB8osM=

**Comment by kolaente** [comment]
Updated to 0.20.2 since that was released in the meantime

**Pull request merged**